### DISCUSSION TITLE
2024/01/08/a-dotnet-parallel-kafka-consumer-proof-of-concept-feat-akkadotnet-and-the-actor-model/

### DISCUSSION THREAD

**Discussion by giscus** [CodingMilitia/Blog#98] [Blog comments]
# 2024/01/08/a-dotnet-parallel-kafka-consumer-proof-of-concept-feat-akkadotnet-and-the-actor-model/

For some time now, I’ve been thinking about implementing a parallel Kafka consumer proof of concept in C#. I finally got around to it, and this post describes the process.

https://blog.codingmilitia.com/2024/01/08/a-dotnet-parallel-kafka-consumer-proof-of-concept-feat-akkadotnet-and-the-actor-model/

**Comment by alefcarlos** [comment]
That's was awesome! 


```
As soon as I started to think about how to implement this, the actor model was the first thing that came to mind - probably because I’ve been trying for years to find a scenario where I could use it 😅
```

I was looking for that as well and I found it on this article!

**Reply by joaofbantunes** [reply]
Glad you enjoyed it 🙂
